### PR TITLE
Rs 2022 04 21

### PR DESCRIPTION
**Pull request by rschlaikjer**
d11 may be short but my god does it have some big ole chunks in it

### CODE REVIEW

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 1]
```diff
@@ -289,27 +312,30 @@ Dead apostles are, after all, no more than just the ruins of what once was a hum
...
 -- 人間の血なんて必要なく、死徒たちと同等、いえそれ以上の超越能力をもつモノたち。
-They don't need human blood for survival, but are otherwise the same as Dead Apostles. No, they're creatures of much greater ability.
+Beings who do not require human blood to survive,
+ and possess transcendental powers on par with―――no, surpassing those of the Dead Apostles.
 }
 [sha:e18904c2fd8b8e67e3d2e7092e35b41c12de719d]{
 -- Page 60, Offset 29669.
 -- 彼女はその真祖と呼ばれる一族の王族なんです。
-Even among 'True Ancestors', she's royalty you see.
+Even amongst True Ancestors, that woman is royalty.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:332]: among the

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 2]
```diff
@@ -569,12 +617,13 @@ They're the dead disciples of a True Ancestor. They're like living painkillers f
 [sha:c6664881369eb3e5af1d223f2a12f4fa6448fdf6]{
 -- Page 78, Offset 29722.
 -- 真祖の中でも特別だった彼女は、自らの意思だけで吸血衝動を抑えられていた。ですが、それも昨夜で限界を迎えた」
-For her, who was special even among Dead Apostles, she could suppress those urges through sheer force of will. However, she reached her limit last night."
+She was special, even among True Ancestors, in that she could suppress those urges through sheer force of will.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:620]: drop commas

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 3]
```diff
@@ -569,12 +617,13 @@ They're the dead disciples of a True Ancestor. They're like living painkillers f
...
+ However, she reached her limit last night."
 }
 [sha:749f2775b733de9e202e8e9de6e1697e3534aa00]{
 -- Page 79, Offset 29723.
 -- 「吸血衝動に果てはありませんし、消える事もない。
-"These impulses have no limit; they will never go away.
+"These impulses are ceaseless; they will never go away.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:626]: Not a fan of ceaseless. maybe "unrelenting"?

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 4]
```diff
@@ -584,7 +633,7 @@ This constant suppression of urges creates sediment, and before long, it overflo
 [sha:da2b800d9e994a0d832413f9147e5d31574b58b1]{
 -- Page 80, Offset 29725.
 -- 「彼らは長く生きれば生きるほど、自らに抱えた吸血衝動を肥大化させていってしまう。
-"The longer they live, the stronger that urge grows.
+"The longer they live, the stronger that urge within them grows.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:636]: revert imo
or change to "the stronger that urge grows within them"

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 5]
```diff
@@ -711,27 +770,29 @@ If she herself knew that she couldn't go on anymore, then what was she doing hun
...
 -- Page 88, Offset 29751.
 -- そんなの―――どう考えたって、ヘンだ。
-No matter how I think about it―――it's weird.
+No matter how I think about it―――it doesn't make sense.
 }
 [sha:d268fa4970f5f081394059dc0a0d633f7b281e7e]{
 -- Page 89, Offset 29752.
 -- 「……先輩の話は、筋が通らない。
-"I don't really get what you're saying.
+"I'm not sure I follow, Senpai.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:783]: drop Senpai imo

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 6]
```diff
@@ -751,42 +812,46 @@ The 12th century, when she was born, was an era of fallen True Ancestors running
...
-"The True Ancestors couldn't leave their fallen counterparts and the army of frivolously created Dead Apostles they'd sired alone.
+"The True Ancestors couldn't leave their fallen counterparts―――and the armies of Dead Apostles they'd sired―――alone.
 }
 [sha:100f40e1f487f98ff780b4b9c3598f2d7fddea29]{
 -- Page 92, Offset 29759.
 -- その手段として、ただ殺す為だけの、それ以外の事を何も必要としない純粋な真祖を誕生させ、処刑役にした」
-So for that――for the sole purpose of killing, without any other duty――a pure True Ancestor was born as an executioner."
+To that end they gave birth to a pure True Ancestor,
+ one who existed for the sole purpose of killing,
+ who could serve as their executioner."
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:822]: maybe "an executioner". "their executioner" sounds like her role is to execute the TAs who sired her (though i guess she _did_ do that).

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 7]
```diff
@@ -826,88 +892,98 @@ I don't know how you could say that chatterbox never spoke.
 [sha:5354e2abd8c1fe7752bea583a1441ff13da7ec48]{
 -- Page 97, Offset 29773.
 -- 昔から―――もう何百年も昔から、彼女はそうやって生きてきた。あの山間の古城に生を享けた時から、ずっと、永遠に変わる事なく」
-Since long ago, since hundreds of years ago, she's lived like that. Since that old castle among the mountains came into being, she has lived like that in perpetuity."
+She's existed like that since antiquity―――for hundreds upon hundreds of years.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:895]: hundreds and hundreds of years

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 8]
```diff
@@ -826,88 +892,98 @@ I don't know how you could say that chatterbox never spoke.
...
-Scenery I shouldn't know, memories I shouldn't be able to see, my field of vision is dizzyingly expanding.
+Images of scenery I could never have known,
+ memories of things I could never have seen,
+ all unfold before my eyes my like a waking dream.
 }
 [sha:677e92dfe7f094e5ed7a6cc00b6e06b028685f18]{
 -- Page 99, Offset 29778.
 -- 山間に広がる最後の楽園。
-Expanding finally to the paradise among the mountains.
+A final paradise, sprawled amongst the mountains.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:927]: "among the mountains" sounds more natural to me

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 9]
```diff
@@ -1048,14 +1128,13 @@ She was just―――
 [sha:15dbc79a7fa91b738edf358b0494b388e738a716]{
 -- Page 112, Offset 29817.
 -- “―――だって、すごく楽しいんだもの。
-// line is from 10_00_ARC10_5
-// following two lines not voiced.
-"―――But, it was really fun.
+// This and subsequent offset from 10_00_ARC10_5, unvoiced
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:1131]: I'd never take a script edit personally but comment edits? this is just crossing a line.

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 10]
```diff
@@ -1130,17 +1218,20 @@ In the end, she was the only one that remained.
 [sha:ecda0335213f7b7620c71e08675132d7e551e7b2]{
 -- Page 117, Offset 29833.
 -- 「同族であり親である真祖たちを処断した後、彼女は自分から城に閉じこもりました。
-"After the elder True Ancestors passed their judgement, she secluded herself in her castle.
+"After executing all of her kin, the True Ancestors that had raised her,
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:1221]: who had raised

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 11]
```diff
@@ -1226,12 +1318,14 @@ She was happy because she had never experienced that trivial repetition until no
 [sha:ea2e895e4055a73dca245bfc870309efb81ff55f]{
 -- Page 124, Offset 29852.
 -- なにが悔しいかって―――あいつ自身、自分がどんなに馬鹿げた道を歩いてきたかを、これっぽっちも感じていない事だ。
-What frustrating convenience―――No matter what foolish path she walked down, she had never felt even that much.
+What's so incredibly frustrating―――is that she herself didn't even have an inkling of how absurd the path she'd been walking was.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:1321]: separating with the cjk dashes like this doesn't work with this new sentence structure. Just drop it

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 12]
```diff
@@ -1380,53 +1475,55 @@ That's just whitewashing the truth. She'd accurately correct it like so:
...
 -- あいつが、自分の命を削って守ったもの。
-What she shaved off her own lifespan to protect.
+The thing she shortened her own lifespan to protect.
 }
 [sha:c537cb6307ac0e2cfe63750ff447c41fd95e45d8]{
 -- Page 135, Offset 29889.
 -- 本来、何よりも優先するべき、自分たちの幸福の在処。
-Naturally, the thing we ought to prioritize over all else: the whereabouts of our own happiness.
+// Assuming 幸福の在処 as the city here
+The thing we naturally ought to prioritize above all else: the place where we found our happiness.
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:1515]: revert "the place where we found our happiness" imo

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 11/11_00_ARC11_2.txt (hunk 9 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 11/11_00_ARC11_2.txt:1131]: Didn't notice that the old comment included following line 
aichanteehee

**Pull request merged**